hack/calculate-cache.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
+# This script calculates the expected content size, actual cached size, and cache-keys used in caching method prior and after
```

Comment:
Can you add the usage of the script?
What is the input and the output?

Reply:
Added some descriptions to `calculate-cache.sh`.